### ISSUE TITLE
Deleted branch idf-release/v4.2

### ISSUE THREAD

**Issue by ul-gh** [espressif/arduino-esp32#5029]
Hi,

as there is now only the master branch which is a pure development version IIRC:

Will there be an arduino-esp32 release supporting IDF v4.2 (current) or IDF v4.3 (upcoming)?

**Comment by me-no-dev** [comment]
Nope. Master branch is tracking current IDF-Master which will become 4.4. BTW it should compile fine against 4.3 as well
